cypress/component/reference/MultipleEntryReferenceEditor.spec.tsx

```diff
@@ -130,4 +136,276 @@ describe('Multiple Reference Editor', () => {
     findLinkExistingBtn().click(); // inserts 1 card
     findLinkExistingBtn().should('not.exist'); // limit reached, button hidden.
   });
+
+  it(`shows status of entries`, () => {
+    const sdk = createReferenceEditorTestSdk({
+      initialValue: [
+        asLink(fixtures.entry.empty),
+        asLink(fixtures.entry.changed),
+        asLink(fixtures.entry.published),
+      ],
+    });
+    mount(<MultipleEntryReferenceEditor {...commonProps} sdk={sdk} />);
+
+    cy.findAllByTestId('cf-ui-entry-card').should('have.length', 3);
+    cy.findAllByTestId('cf-ui-entry-card').eq(0).findByText('draft').should('be.visible');
+    cy.findAllByTestId('cf-ui-entry-card').eq(1).findByText('changed').should('be.visible');
+    cy.findAllByTestId('cf-ui-entry-card').eq(2).findByText('published').should('be.visible');
+  });
+
+  //Do we also want to test that the props are correct, or should that be done in a jest test?
```

Comment:
Any reason not to do it here?

Reply:
It doesn't seem possible to wrap the props inside the `mount()` function, since Cypress throws an error when run through the command line/CI. So we either need to come up with a kind of hacky solution here or do it in a jest test

Reply:
Let's leave a todo. We can check again later